### DISCUSSION TITLE
What is query expression for list documents in http request (postman or swagger)

### DISCUSSION THREAD

**Discussion by theMeysam** [appwrite/appwrite#3341] [Q&A]
Hi
I'm low coder (no coder) person.
I want to get my list of documents by query from postman for example.
I could get all my documents with http request (GET) 

My problem is that I don't know what I should send for query.
(I think need to mentioned I read docs and know there are query expression for some language)

Update:
I use n8n for my http request.

**Comment by brandonroberts** [comment]
Hi @theMeysam 

Each query is done using a REST API. The docs give you the path to the resource you can use in Postman, such as https://appwrite.io/docs/client/database?sdk=web-default#databaseListDocuments for listDocuments

An example would using `http://YOUR_HOSTNAME/v1/database/collections/YOUR_COLLECTION_ID/documents?project=YOUR_PROJECT_ID` as the URL, and providing a header with the `X-Appwrite-Key` that provides an auth key that has access to query documents.

The SDKs manage these things for you on top of the APIs
